modules/acquia_cms_headless/acquia_cms_headless.install

```diff
@@ -33,3 +35,16 @@ function acquia_cms_headless_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Install and update new headless roles.
+ */
+function acquia_cms_headless_update_8001() {
+  \Drupal::configFactory()->getEditable('user.role.headless')->set('label', 'Headless Administrator')->save();
+  // Create Frontend site with preview role.
+  $role = Role::create([
+    'id' => 'frontend_site_with_preview',
+    'label' => 'Frontend site with preview',
+  ]);
+  $role->save();
```

Comment:
@deepakmishra2 This hook is only creating role, but all permissions needs to be added as well. :)

Reply:
@vishalkhode1 changes are done now, can you review this.